_episodes/20-parallel-raster-computations.md

```diff
@@ -0,0 +1,344 @@
+---
+title: "Parallel raster computations using Dask"
+teaching: TODO
+exercises: TODO
```

Comment:
can we estimate 40 minutes for teaching, 20 for exercises? and then revise when this is tested?

Reply:
Sounds good!